docs/explanations/audit-scanner.md

```diff
@@ -0,0 +1,235 @@
+---
+sidebar_label: "Audit Scanner"
+title: "Kubewarden Audit Scanner"
+---
+
+# Kubewarden Audit Scanner
+
+Starting from version `v1.7.0`, Kubewarden introduces a new component called
+the Audit Scanner. This scanner evaluates whether the resources running in the
+cluster comply with the deployed policies. It is particularly useful when
+policies are deployed after the resources. The Audit Scanner allows operators
+to detect resources deployed before the policies that violate the validations
+performed by the policies.
+
+To illustrate the usage of the audit scanner in Kubewarden, let's consider the
+following scenario:
+
+Assume you have deployed a set of policies to enforce specific rules and
+validations on your Kubernetes cluster. However, some resources were deployed
+before these policies were implemented, and you want to ensure that these
+resources comply with the newly deployed policies.
+
+To address this need, Kubewarden introduced the audit scanner component
+starting from version `v1.7.0`. The audit scanner periodically evaluates the
+resources running in your cluster and checks if they comply with the policies
+that have been deployed. By doing so, it helps operators identify any resources
+that violate the validations performed by the policies, even if they were
+deployed before the policies themselves.
+
+See [here](../howtos/audit-scanner) how to enable Audit Scanner
+
+### Policy Reports
+
+When using the Kubewarden Audit Scanner, the results of the policy scans are
+stored in [PolicyReport](https://htmlpreview.github.io/?https://github.com/kubernetes-sigs/wg-policy-prototypes/blob/master/policy-report/docs/index.html) Custom Resource Definitions (CRDs).
+
+:::caution
+Note that the PolicyReport CRDs are under development in the `wg-policy`
+Kubernetes group.Therefore, this documentation can be out of date if a new
+version of the CRDs is released.
+
+Check the `wg-policy` group
+[repository](https://github.com/kubernetes-sigs/wg-policy-prototypes) out for
+more information about the CRDs.
+:::
+
+The audit scanner stores the audit results in the Policy Report Custom Resource
```

Comment:
this first phrase is repeated from the one in line 34, I would drop it, and start with "The PolicyReports CRDs provide.."